js/packages/stream/src/types.ts

```diff
@@ -1,3 +1,13 @@
+import { encode } from "eventsource-encoder";
+
+export enum SSEType {
+  TextDelta = "T",
```

Comment:
lets keep it a little more descriptive
- "text"/"text_delta"
- "tpl_name"
- "tpl_args"/"tpl_args"delta" (lets use args instead of props to keep it a little more generic)
- "context"/"context_delta"
- "message_update" (we can keep it message update and pass a partial message which may or may not include id)